Library/Formula/opendbx.rb

```diff
@@ -0,0 +1,31 @@
+class Opendbx < Formula
+  desc "Extremely lightweight but extensible database access library in C"
+  homepage "http://www.linuxnetworks.de/doc/index.php/OpenDBX"
+  url "http://linuxnetworks.de/opendbx/download/opendbx-1.4.6.tar.gz"
+  sha256 "2246a03812c7d90f10194ad01c2213a7646e383000a800277c6fb8d2bf81497c"
+
+  depends_on "sqlite"
+
+  def install
+    inreplace "utils/Makefile.in", "$(LIBSUFFIX)", ".dylib"
```

Comment:
Nope, just helpful to have it reported so we don't need to do this forever! Thanks.